CODEOWNERS

```diff
@@ -38,3 +38,8 @@
 
 # Docker
 /.circleci/docker/ @jeffdaily @jithunnair-amd
+
+# Github Actions
+# This list is for people wanting to be notified every time there's a change
+# related to Github Actions
+/.github/ @seemethere
```

Comment:
should we add others from the team?

Reply:
Yup, If you'd like to be included on this list please comment below, I'm planning on merging this on Thursday July 15th to give other people time to comment here

Reply:
me

Reply:
me pls

Reply:
i nominate myself